.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -23,7 +23,8 @@ body:
 - type: dropdown
   attributes:
     label: Core Library
-    options: 
+    options:
+      - "MSAL.js v3-alpha (@azure/msal-browser@alpha)" 
       - "MSAL.js v2 (@azure/msal-browser)"
       - "MSAL.js v1 (@azure/msal or msal)"
```

Can we just do this instead? There's a version field below where people can specify

```suggestion
      - "MSAL.js (@azure/msal-browser)"
      - "MSAL.js v1 (@azure/msal or msal)"
```

---

I think the only downside is we won't have different labeling for v3. I don't have a preference either way.

---

If there's a need to differentiate we can update the labeler to read the version field

---

I just don't want to update these templates every time we push a new major version :)